Boa.Constrictor/Screenplay/Conditions/Strings/IsNullOrWhitespaceCondition.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace Boa.Constrictor.Screenplay
+{
+    public class IsNullOrWhitespaceCondition : ICondition<string>
```

Comment:
For readability, please change this class name to `IsNullOrWhitespace`. That mirrors the naming of other Conditions.

Example: `Actor.WaitsUntil(Text.Of(something), IsNullOrWhitespace.Value())`

Reply:
Oops, it is renamed now.